integration-tests/src/tests/client/resharding.rs

```diff
@@ -2021,19 +2021,16 @@ fn test_shard_layout_upgrade_missing_chunks_high_missing_prob_v3() {
 
 // latest protocol
 
-#[ignore]
```

Comment:
feel like it maybe makes sense to send a separate PR for something so unrelated, but i'll allow it :)